private-templates-service/client/src/components/Common/Tags/Tags.tsx

```diff
@@ -124,12 +124,12 @@ class Tags extends React.Component<Props, State>  {
         {tags && tags.map((tag: string) => (
           <Tag ref={(ref: HTMLDivElement) => this.tagRefs[tag] = ref} onAnimationEnd={this.onAnimationEnd} key={tag}>
             <TagText>{tag}</TagText>
-	    {allowEdit &&
-            <TagCloseIcon key={tag} iconName="ChromeClose" onClick={ this.props.tagRemove && (() => this.props.tagRemove!(tag))}/>}
+            {allowEdit &&
+              <TagCloseIcon key={tag} iconName="ChromeClose" onClick={this.props.tagRemove && (() => this.props.tagRemove!(tag))} />}
           </Tag>
         ))}
         {allowAddTag && <AddTagWrapper onSubmit={this.submitNewTag} open={isAdding}>
-          <AddTagInput ref={this.addTagInput} open={isAdding} value={this.state.newTagName} onChange={this.handleChange} onKeyDown={this.onKeyDown} />
+          <AddTagInput ref={this.addTagInput} open={isAdding} value={this.state.newTagName} maxLength={30} onChange={this.handleChange} onKeyDown={this.onKeyDown} />
```

Comment:
I think 30 may be too long, 

![image](https://user-images.githubusercontent.com/17441040/77673621-a46a1c00-6f47-11ea-8a1d-b072b6f7aeb5.png)

this was with 30, maybe try 15?

Reply:
Fixed. Just made the modal larger lol
I don't want the length to be too short, sometimes my file names are long
![image](https://user-images.githubusercontent.com/19780117/77675235-e72cf380-6f49-11ea-9e28-72315a122084.png)